### PR TITLE
Backlog bug by setting defaults

### PR DESCRIPTION
**Pull request by gizzmojr**
Some values may be lowest sortkey

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,6 +121,16 @@
                 border-style: solid;
                 border-color: white;
             }
+            #textBacklog {
+                width: 100px;
+                border-style: solid;
+                border-color: white;
+                border-width: thin;
+                text-align: -webkit-center;
```
Comment (leif81) [inline index.html]: Does it need to be `-webkit` prefixed?

File: index.html [hunk 2]
```diff
@@ -503,6 +519,7 @@
                                 priorities.appendChild(box);
                             }
                         }
+                        lowestPriority = priorityName;
```
Comment (leif81) [inline index.html]: Maybe we should be resetting to default priority, can we determine that somehow?

File: index.html (hunk 1 above)
Comment (gizzmojr) [reply index.html]: fixed

File: index.html (hunk 2 above)
Comment (gizzmojr) [reply index.html]: From my quick research yesterday, there is no default priority query. 

I just found something that will work for this and severity

Comment (gizzmojr) [reply index.html]: Fixed in bbb2d433d21ab76b35fba29099f8b24128fdf78d

File: index.html [hunk 3]
```diff
@@ -1051,6 +1085,41 @@
...
+            }
+
+            function dragBacklogLeave(ev) {
+                if (ev.target.classList.contains("drop-target")) {
+                    ev.target.classList.remove("drop-target-hover");
+                }
+            }
+
             // Register event handlers
             document.getElementById("textProduct").addEventListener("input", function() {
```
Comment (leif81) [inline index.html:1120]: Wondering if we can reuse the existing dragCard functions? Quite a bit seems common.

Comment (gizzmojr) [reply index.html:1120]: Done

**Comment by leif81** [comment]
Hmm I just tried it out with a card. It was in the IN_PROGRESS column, I dragged to Backlog and it changed status to CONFIRMED, but didn't change the milestone to `---`.

**Comment by leif81** [comment]
So `default_milestone` is part of the get product api. I think it's safe to assume that's going to be the backlog.

http://bugzilla.readthedocs.io/en/latest/api/core/v1/product.html#get-product

File: index.html [hunk 4]
```diff
@@ -983,7 +1023,26 @@
...
+                ev.preventDefault();
+
+                var bugData = JSON.parse(ev.dataTransfer.getData("text"));
+
+                var targetStatus = "CONFIRMED";
+                var targetResolution = "";
+                var targetPriority = defaultPriority;
+                var targetSeverity = defaultSeverity;
+
+                httpGet("/rest/product/" + bzProduct + "?type=enterable&include_fields=default_milestone", function(response) {
```
Comment (leif81) [inline index.html]: Maybe we should cache this as part of the sign in to speed it up a bit?

**Comment by leif81** [comment]
Noice :+1:

**Pull request merged**